### ISSUE TITLE
Support for multibranch pipeline projects

### ISSUE THREAD

**Issue by jesperes** [tabeyti/jenkins-jack#70]
As far as I can tell, multibranch projects are not supported. This would be a very nice feature.

**Comment by VeteranCoder** [comment]
I totally agree.  I think most organizations larger than 3 developers would most likely be using multi-branch pipeline - as I do.  This extension is of limited use to me without it.

**Comment by ronanbrowne** [comment]
+1 like idea here but no good without multi branch job support

**Comment by eightnoneone** [comment]
I was really excited to find this extension.
I was really sad when my Pipeline tree was completely empty.

All my pipelines are multi-branch.

I wish I had the chops to do a PR.
